client/src/components/ProductSmallContainer/index.js

```diff
@@ -11,16 +13,26 @@ export class ProductSmallContainer extends Component {
       cards: [],
       ready: false,
     };
-    this.loadResources = this.props.loadResources;
-    this.loadResources(1,3);
   }
 
+  componentDidMount() {
+    loadCard(1, 3).then(result => {
+      this.setState({
+        ready: true,
+        cards: [...this.state.cards, ...result.data.items],
+        loading: false,
+        nextPage: result.data.nextPage
+      })
+    }).catch((error) => {
+      notificationError('Таких товаров не существует', 'Products you\'re looking for is nowhere to be found.', error)
```

Comment:
don't use hardcoded values in components, at least - move them to constants file